src/rdkafka_request.c

```diff
@@ -1878,6 +1878,49 @@ void rd_kafka_ListGroupsRequest_versioned(rd_kafka_broker_t *rkb,
         rd_kafka_broker_buf_enq_replyq(rkb, rkbuf, replyq, resp_cb, opaque);
 }
 
+/**
+ * Send DescribeGroupsRequest supporting a versioned request.
+ */
+void rd_kafka_DescribeGroupsRequest_versioned(rd_kafka_broker_t *rkb,
+                                              const char **groups,
+                                              int group_cnt,
+                                              rd_kafka_replyq_t replyq,
+                                              rd_kafka_resp_cb_t *resp_cb,
+                                              void *opaque) {
+        rd_kafka_buf_t *rkbuf;
+        int i;
+        size_t size;
+        int16_t api_version;
+        size_t groups_arraycnt;
+
+        api_version = rd_kafka_broker_ApiVersion_supported(
+            rkb, RD_KAFKAP_DescribeGroups, 0, 4, NULL);
+
```

Comment:
Should check that api_version is not -1 here (broker doesn't support it at all), see other code.

Reply:
done